backend/homework/5/Question3/src/main/java/org/example/Factor.java

```diff
@@ -0,0 +1,33 @@
+package org.example;
+
+class Factor implements Runnable
+{
+    Thread thread;
+    int number;
+
+    Factor(int number)
+    {
+        this.number = number;
+        thread =new Thread(this,"Factor Thread");
+        Logging.print("\nFactor thread created : "+ thread);
```

Comment:
can't find logging imported in this file

Reply:
did the logging worked here

Reply:
I just add dependency of Logger module to the current module and write <Logger module class name>.<print function> i.e. Logging.print("") and it works.